lms/templates/dashboard/_dashboard_course_listing.html

```diff
@@ -127,18 +127,27 @@ <h3 class="course-title">
             endif
           %>
 
-                % if isinstance(course_date, basestring):
-                    <span class="info-date-block" data-tooltip="Hi">${_(container_string).format(date=course_date)}</span>
-                % elif course_date is not None:
-                    <%
-                        course_date_string = course_date.strftime('%Y-%m-%dT%H:%M:%S%z')
-                    %>
-                    <span class="info-date-block localized-datetime" data-language="${user_language}" data-tooltip="Hi" data-timezone="${user_timezone}" data-datetime="${course_date_string}" data-format=${format} data-string="${container_string}"></span>
+            <span class="info-date-block-container">
+                % if is_unfulfilled_entitlement:
+                    <span class="info-date-block" aria-live="polite">
+                        <span class="icon fa fa-warning" aria-hidden="true"></span>
+                        ${_('You must select a session below to access course.')}
```

Comment:
@HarryRein Queried in https://github.com/edx/edx-platform/pull/16472/files#r153914862 - is this alert necessary?
In any case, it's slightly ungrammatical -> "You must select a session to access the course."